homeassistant/components/habitica/sensor.py

```diff
@@ -291,6 +293,39 @@ def entity_picture(self) -> str | None:
             return f"{ASSETS_URL}{entity_picture}"
         return None
 
+    async def async_added_to_hass(self) -> None:
+        """Raise issue when entity is registered and was not disabled."""
+        if TYPE_CHECKING:
+            assert self.unique_id
+        if entity_id := er.async_get(self.hass).async_get_entity_id(
+            SENSOR_DOMAIN, DOMAIN, self.unique_id
+        ):
+            if (
```

Comment:
I think we should move this to `setup_entry` and combine with my previous comment.